### PR TITLE
Update journey process flow with placeholders & remove check you can use & unnecessary placeholders

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
Amend also the acceptance tests relating to these

### CODE REVIEW

**Comment by JoeChapman** [comment]
Hey @sulthan-ahmed please can you squash those commits and update the commit message?

File: apps/right-to-rent-check/index.js [hunk 1]
```diff
@@ -36,28 +39,34 @@ module.exports = {
       fields: ['living-status'],
       next: '/tenancy-start',
       forks: [{
-        target: '/person-location',
+        target: '/tenant-in-UK',
```
Comment (JoeChapman) [inline apps/right-to-rent-check/index.js]: please don't use mixed-case in the url's. https://www.w3.org/TR/WD-html40-970708/htmlweb.html

File: apps/right-to-rent-check/index.js [hunk 2]
```diff
@@ -36,28 +39,34 @@ module.exports = {
...
         condition: {
           field: 'living-status',
           value: 'no'
         }
       }],
     },
-    '/person-location': {
+    '/tenant-in-UK': {
```
Comment (JoeChapman) [inline apps/right-to-rent-check/index.js]: nope nope nope

File: codecept.conf.js [hunk 3]
```diff
@@ -9,8 +9,9 @@ module.exports = {
   name: 'right-to-rent-check',
   tests: './apps/**/acceptance/features/*.js',
   include: {
+    tenantInUKPage: pagesPath('tenant-in-UK.js'),
```
Comment (JoeChapman) [inline codecept.conf.js]: Please don't use mixed-case filenames. Windows will forgive you if you try to load that file with all lowercase but linux will not.

Comment (sulthan-ahmed) [inline codecept.conf.js]: ahh ok, I was just following what was asked for in the url for the content.  I'll change that

**Comment by sulthan-ahmed** [comment]
OK, I have now squashed the commits

File: apps/right-to-rent-check/acceptance/features/person-in-property.js [hunk 4]
```diff
@@ -21,15 +21,15 @@ Scenario('I see the correct fields on the page', (
   ]);
 });
 
-Scenario('I see errors if I submit a form without selecting anything', (
+Scenario('WHEN I submit a form without selecting anything THEN I see errors', (
```
Comment (JoeChapman) [inline apps/right-to-rent-check/acceptance/features/person-in-property.js]: Consider `When I do not select a field And I submit the form Then I see an error`

File: apps/right-to-rent-check/acceptance/features/person-in-property.js [hunk 5]
```diff
@@ -21,15 +21,15 @@ Scenario('I see the correct fields on the page', (
...
   I,
   personInPropertyPage
 ) => {
   I.submitForm();
   I.seeErrors(personInPropertyPage.livingStatusGroup);
 });
 
-Scenario('I am taken to tenancy start step if I select Yes', (
+Scenario('WHEN I select Yes THEN I am taken to tenancy start page', (
```
Comment (JoeChapman) [inline apps/right-to-rent-check/acceptance/features/person-in-property.js]: Also, in the previous test you make a reference to your interaction with the fields AND the form, i.e. `I submit without selecting...`, but here you only make a reference to your interaction with the field i.e `I select Yes...`

File: apps/right-to-rent-check/acceptance/features/person-in-property.js [hunk 6]
```diff
@@ -38,11 +38,11 @@ Scenario('I am taken to tenancy start step if I select Yes', (
   I.seeInCurrentUrl(tenancyStartPage.url);
 });
 
-Scenario('I am taken to current property step if I select No', (
+Scenario('WHEN I select No & submit the form THEN I am taken to the tenant in uk page', (
```
Comment (JoeChapman) [inline apps/right-to-rent-check/acceptance/features/person-in-property.js]: Use `and`, not `&` because `And` is a thing like `Given`, `When` or `Thena

**Pull request merged**